app/javascript/components/icon/icon-component.jsx

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './icon-styles.scss';
```

Comment:
Is it not good to have some defaults?

Reply:
yeah it's good if we had it, but we always overwrite all the values. I prefer to remove the useless code.